airbyte-config/init/bin/main/config/STANDARD_SOURCE_DEFINITION/9da77001-af33-4bcd-be46-6252bf9342b9.json

```diff
@@ -0,0 +1,8 @@
+{
+  "sourceDefinitionId": "9da77001-af33-4bcd-be46-6252bf9342b9",
+  "name": "Shopify",
+  "dockerRepository": "airbyte/source-shopify",
+  "dockerImageTag": "0.1.0",
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/shopify",
+  "icon": "shopify.svg"
```

Comment:
can you remove the svg from both this JSON and the YAML? there is currently a bug in Airbyte which will cause consumers to break when we include an icon on a new connector. A fix will be delivered but for now it should be removed

Reply:
should the whole block like `"icon": "shopify.svg"` be removed, or just the `"shopify.svg"` ?

Reply:
whole block